ic-utils/src/call.rs

```diff
@@ -199,7 +207,7 @@ impl<'agent, Out> AsyncCaller<'agent, Out>
 where
     Out: for<'de> ArgumentDecoder<'de> + Send + Sync,
 {
-    fn build_call(self) -> Result<UpdateBuilder<'agent>, AgentError> {
+    pub fn build_call(self) -> Result<UpdateBuilder<'agent>, AgentError> {
```

Comment:
Now that this is `pub`, should it have one of those `///` comments?

Reply:
Done. Thanks!